crates/analytics/src/payments/accumulator.rs

```diff
@@ -272,7 +272,14 @@ impl PaymentMetricAccumulator for CountAccumulator {
 }
 
 impl PaymentMetricAccumulator for ProcessedAmountAccumulator {
-    type MetricOutput = (Option<u64>, Option<u64>, Option<u64>, Option<u64>);
+    type MetricOutput = (
+        Option<u64>,
+        Option<u64>,
+        Option<u64>,
+        Option<u64>,
+        Option<u64>,
+        Option<u64>,
+    );
```

Comment:
We should ideally avoid using such tuples with primitive types as it can cause confusion and bugs.

Reply:
should we use a struct then for better clarity and constraint?

Reply:
this refactoring we would likely do separately in a separate pr